doc/input-output-reference/src/energyplus-economics/currencytype.tex

```diff
@@ -0,0 +1,18 @@
+\section{CurrencyType}\label{currencytype}
+By default, the predefined reports related to economics use the \$ symbol. If a different symbol should be shown in the predefined reports to represent the local currency this object can be used. If the object is not used, the \$ sign is used in the reports. Only one CurrencyType object is allowed in the input file.
+
+\subsection{Inputs}\label{inputs-061}
+
+\paragraph{Field: Monetary Unit}\label{field-monetary-unit}
+
+The commonly used three letter currency code for the units of money for the country or region. Based on ISO 4217 currency codes. Common currency codes are USD for the U.S. Dollar represented with `\$' and EUR for Euros represented with `€'. The three letter currency codes can be seen at the following web sites (as of 2008):
```

Comment:
Should this be as of 2018 now?

Reply:
The website shows that standard was last published in August 2015. So I will change it to "August 2015".